Api/Product.php

```diff
@@ -438,7 +438,10 @@ public function setDeliveryStatus(DeliveryStatusEntity $deliveryStatus)
      */
     public function getDeliveryStatus()
     {
-        return new DeliveryStatus($this->entity->getDeliveryStatus(), $this->locale);
+        if(!is_null($this->entity->getDeliveryStatus())){
```

Comment:
````
$deliveryStatus = $this->entitty->getDeliveryStatus();

if (null !== $deliveryStatus) {
    return new DeliveryStatus($deliveryStatus, $this->locale);
}
````

General standards tend towards using `null ===/!==` rather than `is_null` and you should assign method evaluations to variables rather than call them twice or more (it is less code).